understanding/target-size-minimum.html

```diff
@@ -53,330 +53,238 @@
+                        <strong>ユーザエージェントのコントロール:</strong> ターゲットのサイズが<a href="#dfn-user-agent">ユーザエージェント</a>によって決定され、かつコンテンツ制作者によって変更されていない。
                      </li>
                      
                      <li>
-                        <strong>Essential:</strong> A particular <a href="#dfn-presentation">presentation</a> of the target is <a href="#dfn-essential">essential</a> or is legally required for the information being conveyed.
+                        <strong>必要不可欠:</strong> そのターゲットを特定の方法で <a href="#dfn-presentation">提示</a>することが、<a href="#dfn-essential">必要不可欠</a>である、又は伝達される情報に対して法的に要求されている。
                      </li>
                      
                   </ul>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Targets that allow for values to be selected spatially based on position within the
-                        target are considered one target for the purpose of the success criterion. Examples
-                        include sliders, color pickers displaying a gradient of colors, or editable areas
-                        where you position the cursor.
+                     <p class="note-title marker">注記</p>
+                     <p>ターゲット内の位置に基づいて空間的に値を選択できるターゲットは、この達成基準の目的上、一つのターゲットとみなされる。例としては、スライダー、色のグラデーションを表示するカラーピッカー、カーソルを置いた位置を編集できる領域などがある。
                      </p>
                   </div>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>For inline targets the line-height should be interpreted as perpendicular to the flow
-                        of text. For example, in a language displayed vertically, the line-height would be
-                        horizontal.
+                     <p class="note-title marker">注記 </p>
+                     <p>インラインのターゲットの場合、行の高さはテキストのフローに対して垂直方向として解釈されるべきである。例えば、縦書きで表示される言語では、行の高さは水平方向となる。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>要約</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Make controls easier to activate.</dd>
+                  <dt>目標</dt>
+                  <dd>コントロールを作動しやすくする。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Ensure targets meet a minimum size or have sufficient spacing around them.</dd>
+                  <dt>何をすればよいか</dt>
+                  <dd>ターゲットが最低限のサイズを満たしている、又はターゲットの周囲に十分な間隔があることを確認する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>Some people with physical impairments cannot click small buttons that are close together.</dd>
+                  <dt>なぜそれが重要か</dt>
+                  <dd>身体障害のある人の中には、近くにある小さなボタンをクリックできない人もいる。</dd>
```

Comment:
small button**s** that are close **together.** のニュアンスを訳出すると、こんな感じでしょうか。

```suggestion
                  <dd>身体障害のある人の中には、互いに近接している小さなボタン群をクリックできない人もいる。</dd>
```